client.js

```diff
@@ -21,8 +21,10 @@
     ChatWork.prototype.sendMessage = function(params) { 
       var post_data = {
         'body': params.body
+      };
+      if (params.self_unread !== undefined) {
+        post_data['self_unread'] = params.self_unread;
```

Comment:
```suggestion
        post_data.self_unread = params.self_unread;
```
にしたほうがシンプルかなと思いますがいかがでしょうか 😄

Reply:
たしかにですね！ 👍  取り込ませて頂きました